app/assets/javascripts/discourse/app/components/input-tip.gjs

```diff
@@ -9,6 +9,10 @@ const InputTip = <template>
     {{#if @validation.reason}}
       {{icon (if @validation.failed "times" "check")}}
       {{@validation.reason}}
+    {{else}}
+      {{#if @keepSpace}}
+        &nbsp;
```

Comment:
nbsp does feel a bit wrong ... can CSS handle this?

Reply:
Will try